terraform/mtls_gateway/ecr.tf

```diff
@@ -0,0 +1,13 @@
+data "aws_ecr_repository" "mtls_authorizer" {
```

Comment:
Could make a custom TF module out of this with ecr_info object as input and image_uri as output (only support SHA?)